iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/BranchObservability.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Branch;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public interface BranchObservability<B extends Branch<B>> extends Extension<B>, Observability<B> {
+
+    @Override
+    default String getName() {
+        return "branchObservability";
+    }
+
+    /** StandardDeviation for Active Power */
+    float getStandardDeviationP(Branch.Side side);
+
+    BranchObservability<B> setStandardDeviationP(float standardDeviationP, Branch.Side side);
+
+    boolean isRedundantP(Branch.Side side);
+
+    BranchObservability<B> setRedundantP(boolean redundant, Branch.Side side);
+
+    /** StandardDeviation for Reactive Power */
+    float getStandardDeviationQ(Branch.Side side);
+
+    BranchObservability<B> setStandardDeviationQ(float standardDeviationQ, Branch.Side side);
+
+    boolean isRedundantQ(Branch.Side side);
+
+    BranchObservability<B> setRedundantQ(boolean redundant, Branch.Side side);
+
+    /** StandardDeviation for Voltage amplitude */
+    float getStandardDeviationV(Branch.Side side);
+
+    BranchObservability<B> setStandardDeviationV(float standardDeviationV, Branch.Side side);
```

Comment:
@sylvlecl : it is what we decided together but are you sure that V quality has a place in injection and branch observability extensions ? For me it is more for bus bar section and maybe for voltage level (for jonction between several lines without bus bar section) and for configured bus, where we have a V. Furthermore, I have some doubts about observability definition. In operation, the equipments are observable if we have measurements at them (P, Q and V). But as a result of state estimation, what is the good definition of observability ? It seems that it is a definition relying to voltage only. Can you help us a little bit on that ?

Reply:
Sorry for the delay:
- no you're right, we should not have V here, V should only be on bus bars and configured buses, in order to not duplicate it. Note though, that we may drop some values, because some buses have no corresponding bus bar or configured bus.
- well the definition would be that the state estimation had enough information to estimate physical values for that element (which does not necessarily mean that there was some measurement right on that element)